ocaml/xapi/xapi_host.ml

```diff
@@ -2927,4 +2927,7 @@ let apply_updates ~__context ~self ~hash =
   warnings
 
 let set_https_only ~__context ~self ~value =
-  Db.Host.set_https_only ~__context ~self ~value
+  Db.Host.set_https_only ~__context ~self ~value ;
```

Comment:
Please update the field _after_ changing the firewall. That way you don't have to put back the old value in case the script fails.

Reply:
Makes sense, have swapped them around